sdk/storage/azblob/blockblob/responses.go

```diff
@@ -98,3 +98,9 @@ func toUploadReaderAtResponseFromCommitBlockListResponse(resp CommitBlockListRes
 		XMSContentCRC64:     resp.XMSContentCRC64,
 	}
 }
+
+// UploadFileResponse contains the response from method Client.UploadBuffer/Client.UploadFile.
+type UploadFileResponse = UploadReaderAtResponse
```

Comment:
We should rename `UploadReaderAtResponse` to simply `UploadResponse` as we don't have any `UploadReaderAt` API.

Also, can you please take a look at the various response types for the `Upload*` APIs.  E.g. I see a type `UploadReaderAtResponse` which looks to be a copy of a type from the generated code instead of a type alias.

Reply:
There is already a function `func (bb *Client) Upload(ctx context.Context, body io.ReadSeekCloser, options *UploadOptions)` for Upload() API.  `UploadReaderAtResponse` is response received from `uploadReaderAtToBlockBlob` so we can not export it but we it'll be required.

Reply:
Do we actually need to export this though?  Don't we want customers to use the Upload* variants?